langchain/prompts/chat.py

```diff
@@ -164,6 +168,18 @@ class ChatPromptTemplate(BaseChatPromptTemplate, ABC):
     input_variables: List[str]
     messages: List[Union[BaseMessagePromptTemplate, BaseMessage]]
 
+    def __or__(self, other: Any) -> ChatPromptTemplate:
```

Comment:
feels weird to use or for this, would add not make more sense? like adding strings

Reply:
yeah thats fair

Reply:
For piping it seems like it'd make more sense to do it between a prompt and an LLM (basically decompose an llm chain)

```
parsed_result = my_prompt | HuggingFace() | my_parser;
```